winpr/include/winpr/nt.h

```diff
@@ -1288,8 +1288,11 @@ static INLINE NTSTATUS NTSTATUS_FROM_WIN32(long x)
 #undef _FILE_INFORMATION_CLASS
 #undef FileDirectoryInformation
 
+#elif defined(_WIN32)
```

Comment:
Will not work, `MinGW` does not define `WIN32` but it defines `_WIN32`

Reply:
It's _WIN32 already here.